src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="page__calendar-container">
+      <div class="calendar calendar__day--start-day--sun calendar__day--month-length--29">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
We do not need calendar__day--1, calendar__day--2 ...
We use only css to set calendar day number

Reply:
Yup, thanks for help. Rectified everything now, plus noticed, that I made modifier not for calendar, but for calendar__day for some reason. Corrected it also)).